corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/global_navigation_bar.html

```diff
@@ -129,6 +129,14 @@
                     </a>
                 </li>
             {% endif %}
+            {% if user.is_staff %}
```

Comment:
just confirming that this would not fail if there is no `user`? like a non-logged in page.

Reply:
Please refer to [this](https://github.com/dimagi/commcare-hq/pull/34026#discussion_r1466121235) comment.